packages/pluggableWidgets/color-picker-web/src/components/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import { createElement, ReactElement } from "react";
+import classNames from "classnames";
+import { ModeEnum } from "../../typings/ColorPickerProps";
+
+export interface ButtonProps {
+    color: string;
+    mode: ModeEnum;
+    onClick: (hide: boolean) => void;
+    disabled: boolean;
+    hidden: boolean;
```

Comment:
As far as I see, you only pass this `hidden` prop to this component to then pass it to the `onClick` function. This means you don't actually have to pass it into this component. It's better API design to remove this prop and include it directly in the `onClick` prop as you just inline the callback anyways.